### PR TITLE
chore: Bump Newtonsoft.Json

### PR DESCRIPTION
**Pull request by ganeshnj**
*Issue #, if available:*

*Description of changes:*
Combine
https://github.com/aws/aws-lambda-dotnet/pull/1221
https://github.com/aws/aws-lambda-dotnet/pull/1220
https://github.com/aws/aws-lambda-dotnet/pull/1219
https://github.com/aws/aws-lambda-dotnet/pull/1218
https://github.com/aws/aws-lambda-dotnet/pull/1217
https://github.com/aws/aws-lambda-dotnet/pull/1216
https://github.com/aws/aws-lambda-dotnet/pull/1215
https://github.com/aws/aws-lambda-dotnet/pull/1214
https://github.com/aws/aws-lambda-dotnet/pull/1213
https://github.com/aws/aws-lambda-dotnet/pull/1212
https://github.com/aws/aws-lambda-dotnet/pull/1211
https://github.com/aws/aws-lambda-dotnet/pull/1210

and bump corresponding library versions to push new versions.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Libraries/src/Amazon.Lambda.Serialization.Json/Amazon.Lambda.Serialization.Json.csproj [hunk 1]
```diff
@@ -9,15 +9,15 @@
     <AssemblyName>Amazon.Lambda.Serialization.Json</AssemblyName>
     <PackageId>Amazon.Lambda.Serialization.Json</PackageId>
     <PackageTags>AWS;Amazon;Lambda</PackageTags>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>2.0.1</VersionPrefix>
```
Comment (ppittle) [inline Libraries/src/Amazon.Lambda.Serialization.Json/Amazon.Lambda.Serialization.Json.csproj:12]: Newtonsoft.json a transitive dependency for consumers - wont this break binary compatibility?  Thinking we should at least make this a minor bump, though a major bump could also be appropriate.  Thoughts?

Comment (normj) [reply Libraries/src/Amazon.Lambda.Serialization.Json/Amazon.Lambda.Serialization.Json.csproj:12]: Lets bump the version to 2.1.0

Comment (ganeshnj) [reply Libraries/src/Amazon.Lambda.Serialization.Json/Amazon.Lambda.Serialization.Json.csproj]: Updated.

**Review by normj — APPROVED** [review]
Be sure this PR goes out at the same time or after the update to `Amazon.Lambda.Serialization.Json` goes out.

File: Blueprints/BlueprintDefinitions/vs2017/ChatBotTutorial/template/src/BlueprintBaseName.1/BlueprintBaseName.1.csproj [hunk 2]
```diff
@@ -9,6 +9,6 @@
   <ItemGroup>
     <PackageReference Include="Amazon.Lambda.Core" Version="2.1.0" />
     <PackageReference Include="Amazon.Lambda.LexEvents" Version="3.0.0" />
-    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.0.0" />
+    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.0.1" />
```
Comment (ppittle) [inline Blueprints/BlueprintDefinitions/vs2017/ChatBotTutorial/template/src/BlueprintBaseName.1/BlueprintBaseName.1.csproj:12]: Fix blueprints - correct version for Amazon.Lambda.Serialization.Json is `2.1.0`

Comment (ganeshnj) [reply Blueprints/BlueprintDefinitions/vs2017/ChatBotTutorial/template/src/BlueprintBaseName.1/BlueprintBaseName.1.csproj]: Fixed.

**Review by ppittle — APPROVED** [review]

**Pull request merged**